djangocms_audio/cms_plugins.py

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
+from django.utils.translation import ugettext_lazy as _
+
+from cms.plugin_base import CMSPluginBase
+from cms.plugin_pool import plugin_pool
+
+from . import models
+
+
+class AudioPlayerPlugin(CMSPluginBase):
+    model = models.AudioPlayer
+    name = _('Audio Player')
+    allow_children = True
+    child_classes = ['AudioFilePlugin', 'AudioFolderPlugin']
+
+    fieldsets = [
+        (None, {
+            'fields': (
+                'template',
+                'label',
+            )
+        }),
+        (_('Advanced settings'), {
+            'classes': ('collapse',),
+            'fields': (
+                'is_active',
+                'attributes',
+            )
+        })
+    ]
+
+    def get_render_template(self, context, instance, placeholder):
+        return 'djangocms_audio/{}/player.html'.format(instance.template)
```

Comment:
based on the usage of template, I don't think it's the correct name.
I would rename it to theme or something like that because the template is not affected (`player.html`)